FogCreek/Glitch-Community
# Humdrum triangle: VS Code auth page

## Links
* Remix link: https://humdrum-triangle.glitch.me/vscode-auth
* Manuscript link
* Specs/Notion docs: https://www.notion.so/glitch/VS-Code-Auth-Minimal-Langing-Page-a69a523ffd4241ec81a73c3ba14f9aed

## GIF/Screenshots:

## Changes:
* Adds a page that will handle auth for the VS Code extension

## How To Test:
* When signed in you should be prompted to open a VS Code link if you have it installed?
* When signed out you will see a little version of our sign-in
* Confirm the destination after login in localStorage is set to the page

## Feedback I'm looking for:
* Any welcome!
* Did something kind of weird to reuse the sign-in pop without it being a pop

## Things left to do before deploying:
- [ ] I have been testing this with the VS Code team and it seems to be working but let me know if anything is weird
